src/components/TimeseriesDataExport/constants.ts

```diff
@@ -12,8 +12,6 @@ export const defaultStrings = {
   closeBtn: 'Close',
   imageDownloadLabel: 'Image download',
   imageDownloadBtn: 'Download as SVG',
-  cellLimitErr:
```

Comment:
same as cellLimit, since it has been exposed as an interface

Reply:
Also agreed!